frontend/src/entities/configuration/configuration-history-table.tsx

```diff
@@ -34,20 +35,23 @@ export function ConfigurationHistoryTable () {
     });
 
     useMemo(() => {
-        dispatch(listConfigurations({configScope: 'global', numVersions: 10}));
+        dispatch(listConfigurations({configScope: 'global', numVersions: 999999}));
     }, [dispatch]);
 
     const columnDefinition = [
         {
             header: 'Version',
+            sortingField: 'versionId',
             cell: (item) => item.versionId,
         },
         {
             header: 'Changed by',
+            sortingField: 'changedBy',
             cell: (item) => item.changedBy,
         },
         {
             header: 'Change reason',
+            sortingField: 'changeReason',
```

Comment:
I'm not sure it makes sense to sort by changeReason and possibly not changedBy too since there is a search field if you want to look at a specific person or reason. I guess it doesn't hurt to have but could be a little confusing to list things in other than numeric order.

Reply:
by default it filters by version number but it cant hurt to allow users to filter by whatever they want